### DISCUSSION TITLE
Report - (3rd Sem.) Week 2 📢

### DISCUSSION THREAD

**Discussion by denniskp** [SE-TINF22B6/DHBWhub#65] [Weekly Report]
Welcome to our second report of our project. We are excited to share the latest news with you! 🥳

### Documentation and Planning 📃

Let's get the dull news behind us fast.

✅ We agreed on our Software Requirements Specification. For those who are interested, you can find them here:
👉 [DHBWhub Software Requirements Specification](https://github.com/SE-TINF22B6/DHBWhub/blob/master/docs/SOFTWARE_REQUIREMENTS_SPECIFICATION.md) 👈 

✅ We also want to provide you with a rough visualization of the implementation of our website architecture in a UML diagram:

![uml-usecase](https://github.com/SE-TINF22B6/DHBWhub/assets/122602355/a91336ec-7c35-4abc-8a47-5067a4e08b3b)

✅ And for those interested in the more technical side of our project: here is the first version of the ER diagram for our database:

![uml-er](https://github.com/SE-TINF22B6/DHBWhub/assets/122602355/66f090e5-14f6-4f51-9c06-8d563a6aeb04)

### First Designs 🌐

Now it gets interesting. These are first insights in the design for our prototype!

✅ Landing page (user isn't logged in)
![Homepage - PC View](https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/6c3090e6-708c-4bc8-b6cb-4da6bdc3693f)

✅ Homepage (user is logged in)
![Homepage - PC View 2](https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/c9293d65-a7ba-42bf-a131-a22dd6aafd56)

✅ Login windows
<div style="display: flex; flex-direction: row;">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/0e64a018-262f-42b5-812e-8475a8141efb" alt="Login Step 1" width="45%">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/0acde404-1b35-4bb8-99f3-e0e76ca198aa" alt="Login Step 2" width="45%">
</div>

✅ Sign up windows
<div style="display: flex; flex-direction: row;">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/5f676699-8067-41d9-b1bd-ee52b78231b8" alt="Register Step 1" width="33%">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/4a86705e-e200-40a0-a609-b54d2ca0cc7b" alt="Register Step 2" width="33%">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/2ff74e47-6a49-48ac-81ac-cff1a733716f" alt="Register Step 3" width="33%">
</div>

✅ Don't worry, we didn't forget our mobile users 😊
<div style="display: flex; flex-direction: row;">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/37f13aee-504c-4487-9fab-b2c416178b77" alt="Homepage - Smartphone View" style="width: 45%; margin-right: 10%;">
    <img src="https://github.com/SE-TINF22B6/DHBWhub/assets/111356450/2ba0b07f-1ff0-414a-a7b0-d50e3d6b2c4f" alt="Homepage - Smartphone View 2" style="width: 45%;">
</div>

### Sprint 1: What have we achieved ❓  What's to come❔
- ✅Agreeing on the main idea for the project
- ✅Establishing Scrum-platform through GitHub
    - Sprint refinement
    - Categories
    - Templates for tasks
    - Corresponding labels
    - Detailed backlog
- ✅Agreeing on TechStack for now
- ✅Setting up React, Maven and Springboot
- ✅Integrate CI/CD with Dependabot
- ✅Create neat logo with no connotations to other websites
- ✅Creating database and connecting with backend
- ✅Using Figma for initial design for several pages
- 🚧Database schema for user accounts
    -✅ Created first ER-diagram 
    -🚧 Implementation on-going
- 🚧Discussing possible API's 
- 🚧Removing Herobrine 

Until next week 💯

Your DHBWhub Team

**Comment by ghost** [comment]
Cool and intuitive design. Do you also get new notifications when the Profiles are Updated? Looks Great and im looking forward too the final result.

**Comment by LPkkjHD** [comment]
Cool Overview of what you've done the past week! Loving to see the website being implemented with deep links to jroethig as well as some other agile coaching stuff 😂.
